### PR TITLE
Feature/init: Reference arch module for App Gateway

### PR DESCRIPTION
**Pull request by debasish-sahoo-nttd**

### CODE REVIEW

**Comment by debasish-sahoo-nttd** [comment]
A backlog ticket created for tech debt https://dev.azure.com/launch-dso/platform-accelerators/_workitems/create/User%20Story

**Review by ben-vaughan-nttd — APPROVED** [review]

File: sample-tfvars/standard-private-multi-backend.tfvars
Comment (chris11-taylor-nttd) [inline sample-tfvars/standard-private-multi-backend.tfvars:1]: If we're not populating this yet, can we remove this file?

**Review by kevin11-allen-nttd — APPROVED** [review]
lgtm

File: sample-tfvars/standard-private-w-apim-backend-ssl.tfvars
Comment (chris11-taylor-nttd) [inline sample-tfvars/standard-private-w-apim-backend-ssl.tfvars:1]: .tfvars files usually contain the values found in variables, not the variable definitions themselves. Is this intended?

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+      zone_name           = var.custom_private_dns_record["private_dns_zone_name"]
+      resource_group_name = var.custom_private_dns_record["private_dns_zone_rg"]
+      ttl                 = var.custom_private_dns_record["ttl"]
+      records             = [var.private_ip_address]
+    }
+  }
+}
+
+module "application_gateway" {
+  source = "git::https://github.com/launchbynttdata/tf-azurerm-module_primitive-application_gateway.git?ref=feature/init"
```
Comment (chris11-taylor-nttd) [inline main.tf]: Needs to be pointed at a tag, pretty sure this is causing your lint failure as this branch no longer exists.

Comment (debasish-sahoo-nttd) [reply main.tf]: done. sorry about that

File: sample-tfvars/standard-private-w-apim-backend-ssl.tfvars
Comment (debasish-sahoo-nttd) [reply sample-tfvars/standard-private-w-apim-backend-ssl.tfvars:1]: accidentally put that. deleted it

File: sample-tfvars/standard-private-multi-backend.tfvars
Comment (debasish-sahoo-nttd) [reply sample-tfvars/standard-private-multi-backend.tfvars:1]: checked in with contents

**Review by chris11-taylor-nttd — APPROVED** [review]

**Pull request merged**